.lycheeignore

```diff
@@ -1,4 +1,5 @@
 http://your.ip.add.ress:3000/
 http://your.ip.add.ress:3001/
+https://www.bsi.bund.de/
```

Comment:
TFW the german ministry for IT security can't keep its website online :smiling_face_with_tear:

Reply:
That's what you get for putting any trust in a government. ;D